core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/validation/packagers/DockerPackagerValidator.java

```diff
@@ -403,6 +404,10 @@ private static void validateLabels(DockerConfiguration self) {
         if (!self.getLabels().containsKey(LABEL_OCI_IMAGE_REVISION)) {
             self.getLabels().put(LABEL_OCI_IMAGE_REVISION, "{{commitFullHash}}");
         }
+        if (!self.getLabels().containsKey(LABEL_OCI_IMAGE_SOURCE)) {
+            self.getLabels().put(LABEL_OCI_IMAGE_SOURCE, "{{repoUrl}}");
```

Comment:
The value for `{{projectSource}}` remains undefined and thus the template will resolve to an empty string. This is not good. 

If the value should be the url pointing to the git repository then change this to `{{repoUrl}}`.